ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -376,6 +394,51 @@ class GraphsContainer extends React.Component {
                   />
                 ))}
 
+              {changelogData.length > 0 &&
+               <VictoryBar
```

Comment:
Using `VictoryLine` might make more sense here. It's used for highlighting alerts (per above) and that doesn't affect the Y-axis negatively.

In terms of making this easier to click on, I suggest looking at the Victory library docs. It's quite detailed and might give you some different ideas on how best to display this other than vertical lines.